openff/toolkit/topology/molecule.py

```diff
@@ -5508,26 +5638,23 @@ def to_dict(self):
         return return_dict
 
     @property
-    def particles(self):
+    def particles(self) -> Particle:
```

Comment:
```suggestion
    def particles(self) -> Generator[Particle]:
```

Reply:
We're hoping to have this deprecated or removed (I'm not actually sure which). Ref #1303 / #1310

Reply:
Oof, yeah this found a conflict really quickly. This conflicts directly with [this diff](https://github.com/openforcefield/openff-toolkit/pull/1310/files#diff-68cb3f9873e00994ccf957cc089b18cbaf34302d5eeb90c34ca289803e10a69eL5511), where we will no longer allow the user to manipulate virtual sites before parameterization, so there's no sense in talking about `Molecule.particles` because all of the particles must be atoms. Since this never reached the public API, @mattwthompson's changes should take precedence - This conflict doesn't block merging #1310, and this PR should be updated to use `HierarchyElement.atoms` instead of `.particles`.